### PR TITLE
Update Italian language

### PR DESCRIPTION
**Pull request by bovirus**
@maxnet 

Please check and merge. Thanks.

### CODE REVIEW

File: i18n/rpi-imager_it.ts [hunk 1]
```diff
@@ -64,13 +64,13 @@
...
-        <translation type="unfinished"></translation>
+        <source>elimina i dati esistenti nell&apos;unità</source>
+        <translation></translation>
     </message>
     <message>
         <location filename="../downloadthread.cpp" line="277"/>
-        <source>zeroing out first and last MB of drive</source>
-        <translation type="unfinished"></translation>
+        <source>azzera il primo e l'ultimo MB dell&apos;unità</source>
+        <translation></translation>
```
Comment (lurch) [inline i18n/rpi-imager_it.ts:73]: @bovirus I think you were supposed to edit the `translation` tag, rather then editing the `source` tag?!

**Comment by bovirus** [comment]
yes.

Sure.
Sorry fior the error.

Should be


>      </message>

     <message>

         <location filename="../downloadthread.cpp" line="277"/>

-        <source>zeroing out first and last MB of drive</source>

+        <translation>Azzera il primo e l'ultimo MB dell&apos;unità</translation>


I will fix asap.

Thanks. KR.

________________________________
Da: Andrew Scheller <notifications@github.com>
Inviato: giovedì 19 novembre 2020 22:16
A: raspberrypi/rpi-imager <rpi-imager@noreply.github.com>
Cc: Roberto Boriotti <Roberto.Boriotti@canon-europe.com>; Mention <mention@noreply.github.com>
Oggetto: [FROM EXTERNAL] Re: [raspberrypi/rpi-imager] Update Italian language (#120)


@lurch commented on this pull request.

________________________________

In i18n/rpi-imager_it.ts<https://eur02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fraspberrypi%2Frpi-imager%2Fpull%2F120%23discussion_r527203569&data=04%7C01%7Croberto.boriotti%40canon-europe.com%7Cb7b4919611ea4380955608d88cd05406%7Cacbd4e6be8454677853ca8d24faf3655%7C0%7C0%7C637414173670502722%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=Pk9bION0UcjyZVDv2qQdgCkMVy6kMMpEoetANlcbWHE%3D&reserved=0>:

>      </message>

     <message>

         <location filename="../downloadthread.cpp" line="277"/>

-        <source>zeroing out first and last MB of drive</source>

-        <translation type="unfinished"></translation>

+        <source>azzera il primo e l'ultimo MB dell&apos;unità</source>

+        <translation></translation>



@bovirus<https://eur02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fbovirus&data=04%7C01%7Croberto.boriotti%40canon-europe.com%7Cb7b4919611ea4380955608d88cd05406%7Cacbd4e6be8454677853ca8d24faf3655%7C0%7C0%7C637414173670512678%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=VwarRILY2NZ%2FRMehYqVQU5RNjJaj%2B46slZDj1JU%2BzS8%3D&reserved=0> I think you were supposed to edit the translation tag, rather then editing the source tag?!

**Comment by lurch** [comment]
@bovirus You might also want to edit your reply above, to take out your email address!
https://github.com/raspberrypi/rpi-imager/pull/120#issuecomment-730895825

**Comment by bovirus** [comment]
@lurch 

Thanks for the notification.
I remove my email reference.

Thanks.

**Pull request merged**